docs/guide.rst

```diff
@@ -2,4 +2,76 @@
 User Guide
 **********
 
-This user guide provides a walkthrough of the major features in the ``hvpy`` package.
+This guide provides a walkthrough of the major features in the ``hvpy`` package.
+``hvpy`` is a python wrapper for the `Helioviewer API <https://api.helioviewer.org/docs/v2/>`__.
+This is the same API that powers `helioviewer.org <https://helioviewer.org>`__.
+
+Basic Usage
+-----------
+Each API endpoint has a function in ``hvpy`` that shares the same name and has the same input parameters.
+For example if you are looking at `getJP2Image <https://api.helioviewer.org/docs/v2/api/api_groups/jpeg2000.html#getjp2image>`__ in the Helioviewer documentation, you would execute this in ``hvpy`` with the following command:
+.. code-block:: Python
```

Comment:
```suggestion

.. code-block:: Python
```
I am not actually sure if the line break is needed for RST.

Reply:
wait, you meant before

Reply:
Yeah.

Reply:
The line break is needed before.

Reply:
<img width="770" alt="image" src="https://user-images.githubusercontent.com/94071409/186434182-4834cd33-1078-448a-9324-d84c0e144e41.png">

Reply:
Fixed